Journalator/Source/SlashCmd.lua

```
@@ -57,12 +59,117 @@ function Journalator.SlashCmd.Debug(...)
+    end
+
+    -- Check if portfolio exists
+    if not Journalator.InvestONator.GetPortfolio(portfolioId) then
+      local message = "Portfolio " .. portfolioId .. " not found. Use '/jnr invest list' to see available portfolios."
+      Journalator.Utilities.Message(message)
+      return
+    end
+
+    -- Get item ID from name (simplified - would need better item lookup)
+    local itemId = 0 -- This would need proper item ID lookup
+    if Journalator.InvestONator.AddItemToPortfolio(portfolioId, itemId, itemName, amountValue) then
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'Journalator'

(W113)